content/chat/connect.textile

```diff
@@ -0,0 +1,77 @@
+---
+title: Connections
+meta_description: "Manage the realtime connections to Ably."
+product: chat
+languages:
+  - javascript
+---
+
+When you "instantiate":/chat/setup#instantiate a client, a realtime connection is established and maintained with Ably. You can interact with the connection using the @ChatClient.connection@ object in order to monitor a client's connection status.
+
+h2(statuses). Connection statuses
+
+A connection can have any of the following statuses:
+
+|_. Status |_. Description |
```

Comment:
Seems like there's more statuses here than https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/enums/chat_js.ConnectionLifecycle.html

Reply:
Seems we don't offer closing and closed. I guess because there isn't an explicit `close()` method like in pub/sub.